coalib/results/TextPosition.py

```diff
@@ -48,6 +48,30 @@ def column(self):
         return self._column
 
     @classmethod
+    def from_line_col(cls, text, line, column, create_object=True):
+        """
+        Calculates position in a file from line number and column.
+        Uses \\n as the newline character. Position starts from 0.
+
+        :param text:   A tuple of lines in which position is to be calculated.
+        :param line:   The line through which position is identified.
+        :param column: Column through which position is idnetified.
+        :create_object:     Boolean which determines whether to return
+                            a TextPosition Object or a tuple.
+        :return:            a TextPosition object or a tuple of the form
```

Comment:
Start with capital (Also, object is once with a capital O and the second time with a small O)